--- ../../dotdir/.bashrc	2019-06-26 19:08:19.693137185 +0200
+++ laptop	2019-06-26 19:16:34.503115364 +0200
@@ -27,7 +27,7 @@
 alias fgrep='fgrep --color=auto'
 alias egrep='egrep --color=auto'
 
-source /usr/share/bash-completion/bash_completion
+source /etc/bash_completion
 
 ######################################################
 ##################    Custom     #####################
@@ -42,9 +42,10 @@
 # set PATH so it includes user's private bin directories
 PATH="$PATH:$HOME/.local/scripts/:."
 
+alias matlab="wmname LG3D; matlab"
 alias normalize="mp3gain -r *"
 alias ffplay="ffplay -x 1"
-alias calc="bpython -i ~/.local/scripts/calc.py"
+alias calc="bpython3 -i ~/.local/scripts/calc.py"
 
 alias mserver="sshfs dylansavoia@dylansavoia.sytes.net:/var/www/html/Main /media/dylansavoia/server"
 alias umserver="sudo umount /media/dylansavoia/server"
